app/sockets.py

```diff
@@ -0,0 +1,153 @@
+from .__init__ import *
+
+
+from .MyFunctions import *
+# from .models import *
+from .schemas import *
+from .routes import *
+
+
+@socketio.on('join', namespace='/rooms_namespace')
+def on_join(data):
+
+    """
+    verify if attendence is success 
+    then get them join in that room name with otp added
+    now broadcast to that room
+    """
+    try:
+        data = json.loads(data) # now it's dict form, before it was string
+    except:
+        data = {}
+
+    status = data.get('status')
+    if status!='Success':
+        return ''
+    
+    room = '-'.join(data.get('event_name').split(' ')) + '-' + str(data['otp'])
+    username = data.get("username") #add this to payLoad (clientside)
+    join_room(room)
+    mymessage = {
+        'username':data.get('username'),
+        'RoomName':room
+    }
+
+    emit('join_room', json.dumps(mymessage), room=room)
+
+
+
+@socketio.on('admin_stats', namespace='/admin_namespace')
+def made_for_admin():
+
+    """
+    To tell flask-SocketIO that there exsists a namespace named admin_namespace
+    then we can use it in functions written below. Or else error will be thrown.
+    """
+    #admin_stats is mapped with admin_listen
+    pass
+
+
+
+@socketio.on('attendence_request', namespace='/attendence_namespace')
```

Comment:
define smaller endpoint name. Assign them smaller name.

Reply:
How can endpoint be made smaller? their meaning will be lost then.
namespace is added along with them to convey it's a namespace so to ease the dev work to understand that here he is dealing with namespace.

Reply:
Well i wrote this thing because most of the names are too long, but it's okay not an issue. I am resolving this conversation.